src/main/frontend/pipeline-graph-view/pipeline-graph/main/support/SvgStatus.tsx

```diff
@@ -15,122 +13,160 @@ const questionMarkPath =
   "C-0.660,-3.76 -1.21,-3.47 -1.52,-2.87 C-1.69,-2.54 -1.79,-2.07 -1.81,-1.45 L-3.09,-1.45 " +
   "C-3.09,-2.48 -2.80,-3.31 -2.21,-3.94 L-2.21,-3.94 Z";
 
-const hollowCirclePath =
-  "M 0,-6 A 6,6 0 0 1 0,6 A 6,6 0 0 1 0,-6 m 0,1.3 A 4,4 0 0 0 0,4.7 A 4,4 0 0 0 0,-4.7";
-
-const checkMarkPoints =
-  "-2.00 2.80 -4.80 0.00 -5.73 0.933 -2.00 4.67 6.00 -3.33 5.07 -4.27";
-
-const crossPoints =
-  "4.67 -3.73 3.73 -4.67 0 -0.94 -3.73 -4.67 -4.67 -3.73 -0.94 0 -4.67 3.73 -3.73 4.67 0 0.94 " +
-  "3.73 4.67 4.67 3.73 0.94 0";
-
 interface Props {
   result: Result;
   radius: number;
 }
 
+const getImagesPath = () => `${document.head.dataset.imagesurl}`;
```

Comment:
This should resolve the issue with https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/173

I was having a hard time testing this locally, it does work on the local server as is, but I wanted to test with a custom path and had no luck.